README.md

```diff
@@ -1,4 +1,48 @@
-HtUserRegistrationDoctrineORM
-=============================
-
-Doctrine2 ORM storage adapter for HtUserRegistration.
+HtUserRegistrationDoctrineORM
+=============================
+
+This module extends [HtUserRegistration](https://github.com/hrevert/HtUserRegistration)
+by replacing the Zend DB code with Doctrine.
+
+Install
+-------
+
+Require with composer
+```
+composer require hrevert/ht-user-registration-doctrine-orm
+```
+
+Include in config/application.config.php
+```php
+    'modules' => array(
+        ...
+        'HtUserRegistration',
+        'HtUserRegistrationDoctrineORM',  // include after HtUserRegistration
+        ...
+    ),
+```
+
+
+Configure
+----------
+
+Your User entity must implement `HtUserRegistrationDoctrineORM\Entity\UserInterface`.

```

Comment:
I think that this module should be simple as possible which fits every(almost) use case.
If user want anything more, they can extend it to fulfill their needs. If they want `$user->getUserRegistrationRecord();`, they can extend the user entity and add that method. But, I don't see a point in enforcing them to do it.